cw-orch-daemon/src/senders/tx.rs

```diff
@@ -0,0 +1,40 @@
+use std::{future::Future, str::FromStr};
+
+use cosmrs::{bank::MsgSend, tx::Msg, AccountId, Any};
+use cosmwasm_std::Addr;
+
+use crate::{parse_cw_coins, CosmTxResponse, DaemonError};
+
+use super::query::QuerySender;
+
+pub trait TxSender: QuerySender {
+    /// Returns the `AccountId` of the sender that commits the transaction.
+    fn account_id(&self) -> AccountId;
```

Comment:
Does AccountId work with ethereum like addresses(`0x..`)? Just wonder, I'm not sure how good cw-orch with non-bech32 addresses work

Reply:
i don't think it does. I'm not even sure if it should be in here tbh, don't know why nicolas added it...